cypress/utils/apps/query_enhancements/commands.js

```diff
@@ -173,6 +211,7 @@ Cypress.Commands.add(
     } else {
       cy.get('[type="button"]').contains('Cancel').click();
     }
+    cy.wait(1000);
```

Comment:
Can we replace wait with `cy.getElementByTestId('expectedElement').should('be.visible');`?